changelogs/fragments/eos_logging_fix.yaml

```diff
@@ -0,0 +1,2 @@
+- bugfixes:
+    - Fixes eos_logging idempotency issues reported at bug #31862 ( https://github.com/ansible/ansible/pull/40604 )
```

Comment:
The embedded `#` is truncating your message here; just get rid of the `reported at bug #31862` and keep the PR URL (which presumably links to the original issue anyway).